modules/Bio/EnsEMBL/Compara/Locus.pm

```diff
@@ -433,7 +433,7 @@ sub get_sequence {
         # Coordinates are 0-based
         $seq = $faidx_helper->get_sequence2_no_length($self->dnafrag_id, $self->dnafrag_start-1, $self->dnafrag_end-1);
         die "sequence length doesn't match !" if CORE::length($seq) != ($self->dnafrag_end-$self->dnafrag_start+1);
-        reverse_comp(\$seq) if $self->dnafrag_strand < 0;
+        reverse_comp(\$seq) if (defined $self->dnafrag_strand) and ($self->dnafrag_strand < 0);
```

Comment:
You had this change in an early version of #168 and @CristiGuijarro asked why it was needed. So as far as I understand, it's because DnaFragChunk doesn't have the concept of strand ? An alternative is to remove the ambiguity in DnaFragChunk itself and set `dnafrag_strand` to 1 in its constructor.